icepapcms/gui/ui/dialogpreferences.ui

```diff
@@ -587,7 +560,7 @@
    <item row="1" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
This generate a bug when you open the preference, I did a PR: https://github.com/synchrotron-solaris/IcepapCMS/pull/1

Reply:
This is aleready changed in your PR :)

Reply:
yes, thanks!